### ISSUE TITLE
Improve article 2212.07286

### ISSUE THREAD

**Issue by dginev** [dginev/ar5iv#322]
**Exact location of issue**

Minor margin directives are missing.

More interestingly, the paper was given as an example for one needing acessibility support in this stack exchange question:
https://academia.stackexchange.com/questions/193087/screen-readers-for-arxiv-ar5iv

It may be a good starting point to investigate AT offerings and their interplay with the current ar5iv markup.

**Problem details**

Needs simple support for `scrextend.sty`:

```
Conversion complete: 1 warning; 1 error; 1 undefined macro[{addmargin}]; 1 missing file[scrextend.sty]
```

**Comment by dginev** [comment]
Ah, I hadn't realized this is arXiv's accessibility paper. There is already an open issue #306, closing the duplicate.
